go.sum

```diff
@@ -4,6 +4,12 @@ github.com/awans/logio v0.0.9 h1:Fyy3YNMgccHKfwaggcgGI8mO1eNwwrs4bmocte6zBcU=
 github.com/awans/logio v0.0.9/go.mod h1:0faQuZ4AcaTOCePeYWUS07ZsutyNoIYJIuKTMAtVaoI=
```

Comment:
seems to me that some of these dependencies aren't required. Can you clean them up manually and with go tidy?